<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseSettingsDialog</class>
 <widget class="QDialog" name="MouseSettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouse Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mouse Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mouseModeComboBox">
         <property name="toolTip">
          <string>Cursor mode is used to move the mouse cursor
around the screen relative to its current position
depending on how much you move an axis
or if a button is pressed.

Spring mode is used to move the mouse cursor
from the center of the screen depending on how
much you move an axis. The mouse cursor will
be returned to the center of the screen when the
axis is moved back to the dead zone.</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cursor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spring</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Acceleration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="accelerationComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enhanced Precision</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quadratic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cubic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quadratic Extreme</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Power Function</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Mouse Speed Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="changeMouseSpeedsTogetherCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Enable to change the horizontal and vertical speed
boxes at the same time.</string>
          </property>
          <property name="text">
           <string>Change Together</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Horizontal
Speed:</string>
                </property>
                <property name="buddy">
                 <cstring>horizontalSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="horizontalSpinBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>300</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="horizontalSpeedLabel">
              <property name="text">
               <string>1 = 20 pps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Vertical
Speed:</string>
                </property>
                <property name="buddy">
                 <cstring>verticalSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="verticalSpinBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>300</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="verticalSpeedLabel">
              <property name="text">
               <string>1 = 20 pps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="wheelHoriSpeedLabel">
                <property name="text">
                 <string>Wheel Hori.
Speed:</string>
                </property>
                <property name="buddy">
                 <cstring>wheelHoriSpeedSpinBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="wheelHoriSpeedSpinBox">
                <property name="toolTip">
                 <string>Set the speed used for horizontal mouse wheel movement
according to number of simulated notches per second.</string>
                </property>
                <property name="specialValueText">
                 <string/>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="wheelHoriSpeedUnitsLabel">
              <property name="text">
               <string>1 = 1 notch(es)/s</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="wheelVertSpeedLabel">
                <property name="text">
                 <string>Wheel Vert.
Speed:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="wheelVertSpeedSpinBox">
                <property name="toolTip">
                 <string>Set the speed used for vertical mouse wheel movement
according to number of simulated notches per second.</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="wheelVertSpeedUnitsLabel">
              <property name="text">
               <string>1 = 1 notch(es)/s</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Sensitivity:</string>
            </property>
            <property name="buddy">
             <cstring>sensitivityDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sensitivityDoubleSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="smoothingLabel">
            <property name="text">
             <string>Smoothing:</string>
            </property>
            <property name="buddy">
             <cstring>smoothingCheckBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="smoothingCheckBox">
            <property name="toolTip">
             <string>Smoothing can be used to reduce the amount
of jitter that the mouse cursor will exhibit. Using
this option will result in the mouse movement being
slightly less responsive due to changes in the mouse
speed taking longer to occur.</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Spring Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Spring Width:</string>
              </property>
              <property name="buddy">
               <cstring>springWidthSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="springWidthSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Changes the width of the region that the cursor
can move in spring mode. 0 will use the entire width
of your screen.</string>
              </property>
              <property name="maximum">
               <number>16777215</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Spring Height:</string>
              </property>
              <property name="buddy">
               <cstring>springHeightSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="springHeightSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Changes the height of the region that the cursor
can move in spring mode. 0 will use the entire height
of your screen.</string>
              </property>
              <property name="maximum">
               <number>16777215</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MouseSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MouseSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
